packages/plugins/documentation/server/services/helpers/utils/clean-schema-attributes.js

```diff
@@ -202,7 +202,7 @@ const cleanSchemaAttributes = (
             properties: {
               data: {
                 type: 'array',
-                items: componentSchemaRefName.length ? { $ref: componentSchemaRefName } : {},
+                items: componentSchemaRefName.length ? { $ref: componentSchemaRefName + "ListResponseDataItemLocalized"} : {},
```

Comment:
Bit of a nitpick but I'd prefer to be consistent and use string interpolation here instead of concatenation.

```suggestion
                items: componentSchemaRefName.length ? { $ref: `${componentSchemaRefName}ListResponseDataItemLocalized`} : {},
```

Reply:
idk how I can accept the suggested change, but I totally agree

Reply:
nvm, I was on Github Mobile. Fixed now.